packages/errors/src/messages.ts

```diff
@@ -391,6 +393,10 @@ export const SolanaErrorMessages: Readonly<{
     [SOLANA_ERROR__INSTRUCTION_ERROR__UNKNOWN]: '',
     [SOLANA_ERROR__INSTRUCTION_ERROR__UNSUPPORTED_PROGRAM_ID]: 'Unsupported program id',
     [SOLANA_ERROR__INSTRUCTION_ERROR__UNSUPPORTED_SYSVAR]: 'Unsupported sysvar',
+    [SOLANA_ERROR__INSTRUCTION_PLANS__CANNOT_PACK_USING_PROVIDED_MESSAGE]:
+        'Cannot pack the next instructions using the provided message.',
+    [SOLANA_ERROR__INSTRUCTION_PLANS__MESSAGE_PACKER_IS_ALREADY_DONE]:
+        'Failed to pack the next message because the message packer is already done.',
```

Comment:
I wonder if we can make it even more clear what ‘done’ means in this context.
```suggestion
    [SOLANA_ERROR__INSTRUCTION_PLANS__PLAN_ALREADY_COMPLETE]:
        'No more instructions to pack; the message packer has completed the instruction plan',
```

Reply:
This error is very specific to the `MessagePackerInstructionPlan` since aside from that, there isn't another plan that has the concept of being complete. How about `SOLANA_ERROR__INSTRUCTION_PLANS__MESSAGE_PACKER_ALREADY_COMPLETE`?

Reply:
Sure! In any case, the  error message bothered me more than the name of the error code.